client/lib/analytics/ad-tracking.js

```diff
@@ -840,6 +830,35 @@ function recordOrderInFloodlight( cart, orderId ) {
 	recordParamsInFloodlight( params );
 }
 
+/**
+ * Records an order in Facebook (a single event for the entire order)
+ *
+ * @param {Object} cart - cart as `CartValue` object
+ * @param {Number} orderId - the order id
+ * @returns {void}
+ */
+function recordOrderInFacebook( cart, orderId ) {
+	if ( ! isAdTrackingAllowed() || ! isFacebookEnabled ) {
+		return;
+	}
+
+	debug( 'recordOrderInFacebook: Record purchase' );
+
+	const currentUser = user.get();
+
+	const fbParams = {
+		currency: cart.currency,
+		product_slug: cart.products.map( product => product.product_slug ).join( ', ' ),
+		value: cart.total_cost,
```

Comment:
The value _can_ be passed as `0`, so this might need review. I tested the final scenario that you listed where I claimed the free domain. This is what showed up in the console with `value: 0`

![2018-05-02_23-09-08](https://user-images.githubusercontent.com/1563559/39563980-bb9b36e0-4e5e-11e8-8f3b-f8ae7d85157a.png)

Reply:
Thank you! Seems the line that was ignoring 0 value carts disappeared, I have just re-added it.